sdk/Sdk.Generators/Sdk.Generators.csproj

```diff
@@ -20,6 +21,7 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="3.11.0" />
+    <PackageReference Include="System.Text.Json" Version="5.0.2" />
```

Comment:
Analyzers have to package their dependencies manually. See [here](https://github.com/dotnet/roslyn/blob/main/docs/features/source-generators.cookbook.md#use-functionality-from-nuget-packages). Specifically, the section which starts with:

> However, any _generation-time_ dependencies, that is, used by the generator while it is is running and generating code, must be packaged directly alongside the generator assembly inside the generator NuGet package.